worker/src/main/java/gov/cms/ab2d/worker/config/WorkerConfig.java

```diff
@@ -64,8 +63,8 @@
     private int jobQueueCapacity;
 
     @Bean
-    public Executor patientProcessorThreadPool() {
-        final ThreadPoolTaskExecutor taskExecutor = new ThreadPoolTaskExecutor();
+    public ThreadPoolTaskExecutor patientProcessorThreadPool() {
+        final ThreadPoolTaskExecutor taskExecutor = new RoundRobinThreadPoolTaskExecutor();
```

Comment:
Here we'd instantiate the same `ThreadPoolTaskExecutor` class, _but_ with an instance of the custom `BlockingQueue`.